com.unity.shadergraph/Documentation~/HD-Custom-Depth-Node.md

```diff
@@ -0,0 +1,34 @@
+# Custom Depth Node (HDRP)
+
+The Custom Depth Node accesses the custom pass color buffer allocated by HDRP.
+
+## Render pipeline compatibility
+
+| **Node**       | **Universal Render Pipeline (URP)** | **High Definition Render Pipeline (HDRP)** |
+| -------------- | ----------------------------------- | ------------------------------------------ |
+| Custom Depth Node | No                                  | Yes                                        |
+
+## Ports
+
+| Name        | Direction           | Type  | Binding | Description |
+|:------------ |:-------------|:-----|:---|:---|
+| **UV** | Input | Vector 4 | Screen Position | Sets the normalized screen coordinates to sample. |
```

Comment:
| **UV** | Input | Vector 4 | Screen Position | Sets the normalized screen coordinates that this node samples. |